velox/vector/FlatVector.h

```diff
@@ -449,6 +449,15 @@ class FlatVector final : public SimpleVector<T> {
       vector_size_t sourceIndex,
       vector_size_t count);
 
+  // Ensures that '*indices' has space for 'newSize' elements and is mutable.
+  // Sets elements between the old and new sizes to 'initialValue' if the new
+  // size > old size.
+  void resizeValues(
+      BufferPtr* buffer,
+      velox::memory::MemoryPool* pool,
```

Comment:
Why do we need to pass pool? Isn't the vector has it as a member variable?

Reply:
thanks for pointing this out. it was a spillover from my previous change that had this as a generic static function. I have now removed the unnecessary inputs like buffer (use values_) directly and pool (using BaseVector::pool_ directly)